src/test/kotlin/no/nav/helse/ApplicationTest.kt

```diff
@@ -1399,7 +1398,6 @@ class ApplicationTest {
         )
     }
 
-    @Disabled
```

Comment:
Hmm, var det jag som drog in den? :)

Reply:
Husker ikke, men testen var nok deaktivert fordi testet sporadisk feila. 
Ved å annotere riktig JsonFormat på ZoneDateTime på Cache, løser det problemet.

Reply:
Aaaaah, nice catch !